fpga_interchange/fpga_interchange.cpp

```diff
@@ -345,6 +410,15 @@ void FpgaInterchange::write_physical_netlist(const Context * ctx, const std::str
 
     for(auto & cell_name : placed_cells) {
         const CellInfo & cell = *ctx->cells.at(cell_name);
+
+        if(cell.bel == BelId()) {
+            continue;
+        }
+
+        if(!ctx->isBelLocationValid(cell.bel)) {
```

Comment:
this seems more like some kind of error would be appropriate than just skipping over the cell?

Reply:
Fair enough.

Reply:
Fixed.